polling/frequent/activities.py

```diff
@@ -17,6 +19,17 @@ async def compose_greeting(input: ComposeGreetingInput) -> str:
     while True:
         try:
             result = test_service.get_service_result(input)
+            activity.logger.info(f"Exiting activity ${result}")
             return result
-        except Exception:
+        except Exception as e:
+            # swallow exception since service is down
+            activity.logger.error(e)
```

Comment:
I don't think you should log the exception at the error level if it's expected to keep happening, consider:

```suggestion
            activity.logger.debug("Failed, trying again shortly", exc_info=True)
```